Scripts/PlayerController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+namespace Outscal.BasicUnity2DProject
+{
+    public class PlayerController : MonoBehaviour
+    {
+        [SerializeField] private float timer = 0.2f;

```

Comment:
Use spaces and header  here so that you will not be confused in your own variables